examples/local.py

```diff
@@ -2,3 +2,4 @@
 SERVER = '192.168.0.33'
 SSID = 'MY_WIFI_SSID'
 PW = 'PASSWORD'
+PORT = 8123
```

Comment:
local.py should contain a port configuration to make it more obvious, that it can be changed. Also people might run a mqtt broker on their machine and if they want to run the examples, they shouldn't have to change the port in the code first. This should be easy.